src/main/resources/db/migration/V42_6__Create_post_table.sql

```diff
@@ -0,0 +1,11 @@
+create table if not exists post
+(
+    id VARCHAR(255) PRIMARY KEY,
+    title VARCHAR(255),
+    content TEXT,
+    creation_datetime TIMESTAMP,
+    last_update_datetime TIMESTAMP
+);
+create index if not exists post_id_index on post(id);
```

Comment:
in postgresql the PRIMARY KEY have automaticly an index. so u don't need to add that